### ISSUE TITLE
Feature request : webauth

### ISSUE THREAD

**Issue by BirdInFire** [faldez/tanoshi#301]
I would like to ask for Webauth (passwordless) as replacement for the password, thanks.

**Comment by faldez** [comment]
Wouldn't it be overkill for a small app such as tanoshi? What I have in mind is implement SSO or 2FA

**Comment by BirdInFire** [comment]
> Wouldn't it be overkill for a small app such as tanoshi? What I have in mind is implement SSO or 2FA

Actually passwordless are the future (and webauth protect again phishing, but i have open the FR to have it in the roadmap i don't had any idea to see it append before 2FA

**Comment by faldez** [comment]
I agree passwordless is the future, more importantly with apple launching passkey.

I just don't know if it's worth the effort for this app

**Comment by faldez** [comment]
I just take a look at webauthn, and it's really cool. I'll look into it

**Comment by FrancisAgostini** [comment]
not sure if it would be a different suggestion but if it could work with authelia as well would be amazing!

**Comment by HellsCrimson** [comment]
Authelia works with a reverse proxy before a service so it should already work with tanoshi.
